packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -20,6 +20,66 @@ const uniqueRoute = (route) => {
   return `${route}-${routeCount}`
 }
 
+describe('network stubbing - not skipped', () => {
+  it('stops waiting when an xhr request is canceled', () => {
+    cy.visit('http://localhost:3500/fixtures/generic.html')
+
+    cy.intercept('POST', /users/, {
+      body: { name: 'b' },
+      delay: 2000,
+    }).as('createUser')
+
+    cy.window()
+    .then((win) => {
+      const stub = cy.stub()
+
+      const xhr = new win.XMLHttpRequest()
+
+      xhr.open('POST', '/users/')
+
+      xhr.abort = stub // this should not get called
+      xhr.onerror = stub // this should not fire
+      xhr.onload = stub // this should not fire
```

Comment:
Could/should these be actual assertions that they're not called?

Reply:
honestly? I should see if i can remove those. thats testing browser behaviour we're not really concerned with. I copied this over from the xhr system test.